src/scalper_utils.ts

```diff
@@ -223,12 +217,14 @@ export async function getSpotDelta(
     }
     spotDelta = mainDelta + tokenDelta + spotDelta;
   }
-  const openBookOO = await market.findOpenOrdersAccountsForOwner(
-    connection,
-    owner.publicKey,
-  );
-  for (const openOrders of openBookOO) {
-    openOrderQty += openOrders.baseTokenTotal.toNumber() / 10 ** tokenDecimals;
+  if (market !== undefined) {
```

Comment:
Is an undefined market expected? Seems like we shouldnt be calling this if there isnt a market.

Reply:
For BTC there isn't a OpenBook mkt

Reply:
Being added soon to mango, premeptively adding